packages/veui/src/components/Table/_TableHead.js

```diff
@@ -1,14 +1,36 @@
 import Checkbox from '../Checkbox'
 import Sorter from './_Sorter'
+import Popover from '../Popover'
 import prefix from '../../mixins/prefix'
 import table from '../../mixins/table'
 import i18n from '../../mixins/i18n'
 import '../../common/uiTypes'
+import { isFocusable } from '../..//utils/dom'
 
 export default {
   name: 'veui-table-head',
+  components: {
+    'veui-popover': Popover
+  },
   mixins: [prefix, table, i18n],
   uiTypes: ['transparent'],
+  data () {
+    return {
+      openMap: {}
+    }
+  },
+  methods: {
+    handleMouseEnter (id, e) {
+      if (e.target && isFocusable(e.target)) {
+        this.$set(this.openMap, id, false)
+      } else {
+        this.$set(this.openMap, id, true)
+      }
```

Comment:
```suggestion
      if (!isFocusable(e.target)) {
        this.$set(this.openMap, id, false)
      }
```

1. `e.target` 不可能没有吧。
2. `true` 的情况根本不需要设置吧。